From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tommi Kenakkala <tommi.kenakkala@oss.tieto.com>
Date: Mon, 16 Jun 2014 16:48:33 +0300
Subject: [PATCH] Add O2 UK MMS settings

There was no MMS settings for O2 UK so adding an entry of some
settings that were found. Previously there were some O2 settings
under GiffGaff UK MVNO section.
---
 serviceproviders.xml | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/serviceproviders.xml b/serviceproviders.xml
index 55a09a29b..a44b3a6a3 100644
--- a/serviceproviders.xml
+++ b/serviceproviders.xml
@@ -5671,8 +5671,10 @@ conceived.
 				<plan type="postpaid"/>
 				<usage type="internet"/>
 				<name>Contract</name>
+				<protocol type="ip"/>
 				<username>o2web</username>
 				<password>password</password>
+				<authentication method="pap"/>
 				<dns>193.113.200.200</dns>
 				<dns>193.113.200.201</dns>
 			</apn>
@@ -5715,6 +5717,15 @@ conceived.
 				<username>o2wap</username>
 				<password>password</password>
 			</apn>
+			<apn value="wap.o2.co.uk">
+				<usage type="mms"/>
+				<name>O2 UK Pay Monthly MMS</name>
+				<username>o2wap</username>
+				<password>password</password>
+				<authentication method="pap"/>
+				<mmsc>http://mmsc.mms.o2.co.uk:8002</mmsc>
+				<mmsproxy>82.132.254.1:8080</mmsproxy>
+			</apn>
 		</gsm>
 	</provider>
 	<provider>
